.github/ISSUE_TEMPLATE/stable_feature.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Stable Feature Requirements
+about: Requirements for promoting a feature to stable
+---
+
+This page lists the requirements for promoting a feature to stable. Please check off and document the steps as they are completed.
```

Comment:
On each of these we should note that they include all requirements for the previous step. So maybe in stable there is a place to put a link to the issue for beta, etc. So we can chain all they way back and see all the items have been checked off?

One thing to think about is how to manage growing the checklist while features are in the process. If we add a new item to beta checklist, but a feature is already beta, how do we make sure they did it before going to stable?